x-pack/winlogbeat/module/powershell/test/testdata/600.evtx.golden.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "@timestamp": "2020-02-26T09:38:42.4738478Z",
+    "event": {
+      "action": "Provider Lifecycle",
+      "code": 600,
+      "kind": "event",
+      "module": "powershell",
+      "provider": "PowerShell",
+      "sequence": 11
+    },
+    "host": {
+      "name": "vagrant-2019"
+    },
+    "log": {
+      "level": "information"
+    },
+    "message": "Provider \"Variable\" is Started. \n\nDetails: \n\tProviderName=Variable\n\tNewProviderState=Started\n\n\tSequenceNumber=11\n\n\tHostName=ConsoleHost\n\tHostVersion=5.1.17763.1007\n\tHostId=025881f8-9f17-4088-ade7-b7ceb4f82115\n\tHostApplication=powershell.exe -EncodedCommand CgAJAAkAJAB0AG0AcABfAGYAaQBsAGUAXwBwAGEAdABoACAAPQAgAFsAUwB5AHMAdABlAG0ALgBJAE8ALgBQAGEAdABoAF0AOgA6AEcAZQB0AEYAdQBsAGwAUABhAHQAaAAoACIAJABlAG4AdgA6AFQARQBNAFAAXAB3AGkAbgByAG0AYwBwAC0ANwA5ADYAYwAzADEANwA3AC0AMwBiADMANAAtADQAOABlADUALQA3AGQAMgAwAC0AZABiADQAYQA5ADUAYwAzADMAZQBiADYALgB0AG0AcAAiACkACgAJAAkAaQBmACAAKABUAGUAcwB0AC0AUABhAHQAaAAgACQAdABtAHAAXwBmAGkAbABlAF8AcABhAHQAaAApACAAewAKAAkACQAJAFIAZQBtAG8AdgBlAC0ASQB0AGUAbQAgACQAdABtAHAAXwBmAGkAbABlAF8AcABhAHQAaAAgAC0ARQByAHIAbwByAEEAYwB0AGkAbwBuACAAUwBpAGwAZQBuAHQAbAB5AEMAbwBuAHQAaQBuAHUAZQAKAAkACQB9AAoACQA=\n\tEngineVersion=\n\tRunspaceId=\n\tPipelineId=\n\tCommandName=\n\tCommandType=\n\tScriptName=\n\tCommandPath=\n\tCommandLine=",
+    "powershell": {
+      "process": {
+        "executable_version": "5.1.17763.1007",
+        "is_remote": false
+      },
+      "provider": {
+        "name": "Variable",
+        "new_state": "Started"
+      }
+    },
+    "process": {
```

Comment:
I am treating `Host` (which is the host application where PS engine gets executed) as it is the process executed here. Could not think of a good alternative to map it to ECS, other than adding them to the custom `powershell` namespace. WDYT?